dotCMS/src/main/java/com/dotcms/publisher/business/PublishQueueElementTransformer.java

```diff
@@ -113,6 +116,10 @@ private static Map<String, Object> getMap(final String id, final String type,
         return result;
     }
 
+    private static boolean isCategory(final String type) {
+        return type.equals(PusheableAsset.CATEGORY.getType());
```

Comment:
probably the constant should be at the left side such as 

PusheableAsset.CATEGORY.getType().equals(type)

Reply:
really I dont need this method anymore https://github.com/dotCMS/core/pull/23183/commits/efcc202067d1ebc9384341b6869f6b78aa350bc7#diff-3389c7193d609b29dbc719b8c559e5149211bb7b105aa9bee2eb84b1c7f733b1L119